src/main/AndroidManifest.xml

```diff
@@ -43,6 +43,14 @@
         <activity android:name=".ui.preferences.MainActivity"
                   android:label="@string/app_name"/>
 
+        <activity android:name=".ui.bubbleservice.SharedActivity">
+            <intent-filter>
```

Comment:
We should change the label for this activity to `Active Bubble Translate`, then the user can understand the action that he will do